.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -44,30 +45,14 @@ body:
     validations:
       required: false
 
-  - type: input
-    id: platform-version-if-selfhosting
-    attributes:
-      label: If you're self-hosting Renovate, tell us what version of the platform you run.
-      description: |
-        This field is for the version number of your _platform_, so one of these:
-          - AWS CodeCommit
-          - Azure DevOps
-          - Bitbucket Cloud
-          - Bitbucket Server
-          - Azure DevOps
-          - Gitea or Forgejo
-          - GitHub (.com and Enterprise Server)
-          - GitLab (.com and CE/EE)
-    validations:
-      required: false
-
   - type: dropdown
     id: regression-error
     attributes:
       label: Was this something which used to work for you, and then stopped?
       options:
+        - 'Placeholder value, please select the correct response from the dropdown'
+        - 'I\'m trying to get this working for the first time'
```

Comment:
I'm still get the warning message when looking at the "fixed" file on GitHub. 😞 

Maybe the `'I\'m` part is causing problems?

https://github.com/renovatebot/renovate/blob/chore/21034-issue-templates/.github/ISSUE_TEMPLATE/bug_report.yml

Reply:
Seems like it's fixed now